### DISCUSSION TITLE
about/

### DISCUSSION THREAD

**Discussion by giscus** [jeremy-feng/giscus#3] [Announcements]
# about/

https://fengchao.pro/about/

**Comment by yanedie** [comment]
我给你发了邮件，结果显示邮件地址无效

**Reply by jeremy-feng** [reply]
你好，我这边看到的邮箱是正确的，你可以再检查看看～

**Comment by fan2goa1** [comment]
您好，请问一下这个blog的模板有嘛？如果有的话有无开源地址？我最近也想做一个自己的blog，觉得这个模板很好看，想参考一下，谢谢！

**Reply by jeremy-feng** [reply]
你好，谢谢你的喜欢！

你可以参考 Material for MkDocs：

- 项目主页：[https://squidfunk.github.io/mkdocs-material/](https://squidfunk.github.io/mkdocs-material/)
- GitHub 仓库：[https://github.com/squidfunk/mkdocs-material](https://github.com/squidfunk/mkdocs-material)

**Reply by fan2goa1** [reply]
谢谢你，祝学业顺利！

**Comment by shenweiyan** [comment]
很棒的站点

**Reply by jeremy-feng** [reply]
谢谢喜欢❤️

**Comment by NuanBing** [comment]
vocal，发现DSBA学长了！

**Reply by jeremy-feng** [reply]
:eyes:

**Comment by Wcowin** [comment]
很棒的个人网站，而且是我见过的为数不多的 mkdocs站点。希望我的网站加你一个[友链](https://wcowin.work/about/link.html)

**Reply by jeremy-feng** [reply]
谢谢反馈和认可 :heart: 欢迎添加~

**Comment by shenweiyan** [comment]
请教一下博主搜索框的 placeholder （**按 F 或 S 搜索**）是怎么实现的？谢谢。

**Reply by jeremy-feng** [reply]
在 `overrides/partials/languages/zh.html` 中加入 `search.placeholder` 的翻译即可修改占位符。

```
{% macro t(key) %}{{ {
  "search.placeholder": "按 F 或 S 搜索",
}[key] }}{% endmacro %}
```

**Reply by shenweiyan** [reply]
又学习到了，谢谢👍

**Comment by changyifangyan** [comment]
实习中用到了时间序列交叉验证，在这里找到了一篇非常详细的文章。读完觉得写得很好，看到作者头像才发现竟然是经院的学长，真是有缘

**Reply by jeremy-feng** [reply]
谢谢你的鼓励 ❤️ 也很幸运在这儿遇见经院学弟/妹（看来一直用本科毕业照做头像还有这个好处╰(○'◡'○)╮

**Comment by jcfllx** [comment]
想请教一下，请问如何建立这样的知识分享主页呢？是用什么工具做, R-markdown还是vscode还是什么quarto?搞不太懂这些工具的区别， 我不太懂R，但是懂python，不知道用哪个比较好，如果可以欢迎邮件回复，谢谢! 1919142599@qq.com

**Reply by jeremy-feng** [reply]
你可以参考 Material for MkDocs：https://squidfunk.github.io/mkdocs-material/

许多知识分享的站点都是基于 Markdown 文件构建的静态网页，因此会 Markdown 语法就完全足够。会 Python 可以帮助你实现更高阶的自定义功能。

另外，推荐你多使用 AI 帮你完成选型、快速搭建 Demo，相信你体验后会有自己的认知和判断~

**Comment by Charlie-ChenLin** [comment]
cool! 为什么从quant转ai了，求心路历程分享

**Reply by jeremy-feng** [reply]
因为热爱～因为想改变世界，哪怕一点点 :grinning:

https://fengchao.pro/blog/at-a-crossroads-at-24/

**Comment by CXpeerless** [comment]
网上冲浪偶然发现，感觉真的很不错，持续关注
